pycode/epidemiology/epidata/transformTwitterData.py

```diff
@@ -0,0 +1,76 @@
+#############################################################################
+# Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+#
+# Authors: Martin J. Kuehn
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+import os
+import pandas as pd
+
+import getDataIntoPandasDataFrame as gd
+import defaultDict as dd
+import geoModificationGermany as geoger
+
+
+def transformTwitterData(file_format=dd.defaultDict['file_format'],
+                         out_folder=dd.defaultDict['out_folder'].split(
+    '/pydata')[0]
```

Comment:
this is breaks very easily. the default directory could easily change and not contain pydata at all. pydata could be in the path multiple times. the path could use other separators. It's also very inconsistent, since out_folder is different here from every other function.

mobility didn't use to be required for data downloading. since it is now, we need to handle directories differently. either include mobility in the expected directory structure and use the parent directory of pydata and mobility as out_folder. or pass in the path to mobility as an argument. the mobility folder argument could have a default argument similar to out_folder, but default arguments for directories are very unreliable and probably should never be used (other than the current directory). I would like to get rid of the default out_folder entirely. There are so many ways in which this breaks...

The same thing is done in getMobilityData.